content/_index.md

```diff
@@ -1,20 +1,27 @@
-The Virtual Observatory (VO) is the vision that astronomical datasets and other
-resources should work as a seamless whole. Many projects and data centres
-worldwide are working towards this goal. The International Virtual Observatory
-Alliance (IVOA) is an organisation that debates and agrees the technical
-standards that are needed to make the VO possible. It also acts as a focus for
-VO aspirations, a framework for discussing and sharing VO ideas and technology,
-and body for promoting and publicising the VO.
+The Virtual Observatory (VO) is an international research data infrastructure
```

Comment:
Should we spell out here the difference between the VO and the IVOA and describe their relationship?

In other words, the VO is comprised by the body of standards, and the IVOA debates and implements those standards.

The original text I think does an okay job of making this distinction but could use some restructuring/rewording.